next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  serverRuntimeConfig: {
+    apiUrl: "https://construct-hub-testing.dev-tools.aws.dev"
+  }
+}
```

Comment:
@MrArnoldPalmer @eladb This `next.config.js` file would be a good candidate for how we can allow `construct-hub` to set server & runtime configurations on the webapp. 

We can define `serverRuntimeConfig` and `publicRuntimeConfig` properties with the following behaviors:
- `serverRuntimeConfig` is only exposed on the server
- `publicRuntimeConfig` is exposed across client / server, meaning it's perfect for concepts such as custom themes / branding

Reply:
this is interesting, with the current layout though construct hub doesn't "build" the frontend, and so can't pass build time values. By the time it has access to construct-hub-webapp files, they are already built. This may be more argument for collapsing the two.